jdaviz/configs/imviz/helper.py

```diff
@@ -620,3 +817,14 @@ def link_image_data(app, link_type='pixels', wcs_fallback_scheme='pixels', wcs_u
             app.data_collection.set_links(links_list)
         app.hub.broadcast(SnackbarMessage(
             'Images successfully relinked', color='success', timeout=8000, sender=app))
+
+
+def _next_subset_num(label_prefix, subset_groups):
+    """Assumes ``prefix i`` format.
+    Does not go back an fill in lower but available numbers. This is consistent with Glue.
```

Comment:
```suggestion
    Does not go back and fill in lower but available numbers. This is consistent with Glue.
```

We'll need more advanced logic soon once if/when supporting renaming subsets, but since we don't (officially) yet, this is probably fine.